training/src/anemoi/training/commands/mlflow.py

```diff
@@ -131,5 +169,55 @@ def run(args: argparse.Namespace) -> None:
             ).sync()
             return
 
+        if args.subcommand == "prepare":
+            import mlflow
+            from hydra import compose
+            from hydra import initialize
+            from omegaconf import OmegaConf
+
+            from anemoi.training.diagnostics.mlflow.client import AnemoiMlflowClient
+
+            # Load configuration
+            with initialize(version_base=None, config_path="./"):
+                cfg = compose(config_name=args.config_name)
+
+            # Create MLflow client and get experiment
+            client = AnemoiMlflowClient(cfg.diagnostics.log.mlflow.tracking_uri, authentication=True)
+            experiment_id = client.get_experiment_by_name(cfg.diagnostics.log.mlflow.experiment_name).experiment_id
```

Comment:
I think this might fail, if the experiment does not exist, so we could have some functionality to check if the experiment exist and if not create it ahead of the run_id generation?

Reply:
Done.